deploy-as-code/helm/charts/core-services/egov-url-shortening/values.yaml

```diff
@@ -62,6 +62,10 @@ env: |
     value: "false"  
   - name: MANAGEMENT_SECURITY_ENABLED
     value: "false"
+  - name: HASHIDS_SALT
+    value: {{ index .Values "egov-url-shorten-hashid-salt" | quote}}
+  - name: HASHIDS_MIN_LENGTH
+    value: {{ index .Values "egov-url-shorten-hashid-min-length" | quote}}
```

Comment:
To maintain uniformity, append _S_ to the variable name _hashid_. The the variable name would become **hashids-min-length**